examples/nlp/gpt/conf/inference_rm.yaml

```diff
@@ -6,12 +6,20 @@ trainer:
   precision: bf16 # 16, 32, or bf16
 
 inference:
-  micro_batch_size: 4
   port: 5555 # the port number for the inference server
+  model_name: "reward_model"
+  max_queue_delay_microseconds: 2000
+
+  # this should be set at the model level
+  inference_micro_batch_size:
```

Comment:
f90f4d66db748803f454af80ac9bc035a7d31809 I've documented this on the change log. I'm not too particular about this naming but decided to change this to inference_micro_batch_size so it's consistent with the naming of PPO critic. But if you think micro_batch_size is better I can change it back

Reply:
Actually given your explanation on what this setting is used for, I wonder if maybe a better name would be `pytriton_preferred_batch_size` (matching the `preferred_batch_size` name from the PyTriton doc you linked to).

My concern with `inference_micro_batch_size` is that it sounds like this is the mbs used at inference in the model (though the comments clarify it, so I don't feel too strongly about it either).

Reply:
unless you feel very strongly about this i think we should keep this naming. here's my rationale

1. this prevents another breaking change in the critic
2. inference micro batch size is more general -> if we ever consider doing a serverless rm inference script then we can use the same argument even when we're not using pytriton. I think this might hard code our implementation too much with the variable name.


on a side note: I did have this uncleaned [script](https://github.com/NVIDIA/NeMo-Aligner/blob/1ed3c46f5d5f565d81f9ac9e991e4aaf600290f0/examples/nlp/gpt/run_reward_model.py) that runs the reward model, but i decided against putting that on main

Reply:
Thanks, I see -- I thought this was a very PyTriton-specific setting. Ok to keep `inference_micro_batch_size` then.